extensions/github/src/components/DiscussionListItem.tsx

```diff
@@ -0,0 +1,68 @@
+import { List, Icon, Color, ActionPanel, Action, Image } from "@raycast/api";
+
+import { DiscussionFieldsFragment } from "../generated/graphql";
+
+function getDiscussionIcon(discussion: DiscussionFieldsFragment):
+  | {
+      value: Image.ImageLike | undefined | null;
+      tooltip: string;
+    }
+  | Image.ImageLike
+  | undefined {
```

Comment:
```suggestion
function getDiscussionIcon(discussion: DiscussionFieldsFragment): List.Item.Props["icon"] {
```

Reply:
Nice. Didn't know that this is possible in TS.